<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui::DJContext</class>
 <widget class="QWidget" name="Ui::DJContext">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background: #4d4c4d;
    font-family: handelgotdlig;
    font-weight: 200;
}



 QComboBox {
     border: 1px solid #141414;
     border-radius: 4px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
     color: #e8e8e8;
 }

 QComboBox:editable {
    background: #5b5a5b
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
      stop: 0 #3c3c3c, stop: 1 #262627);
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: #5b5a5b
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

QComboBox QAbstractItemView {
    border: 1px solid #141414;
    selection-background-color: #5b5a5b;
}

 QSpinBox {
     /*padding-right: 15px; make room for the arrows */
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
    color: #e8e8e8;
     border: none;
     border-radius: 4px;
     /*border-width: 3;*/
}

QSpinBox::up-button {
    border: none;
}

QSpinBox::down-button {
    border: none;
}

QLineEdit {
    border: 1px solid #141414;
    border-radius: 4px;
    color: #e8e8e8;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #3c3c3c, stop: 1 #262627);
    selection-background-color: #e8e8e8;
    selection-color: #262627
}

QProgressBar {
    border: 1px solid #141414;
    border-radius: 4px;
    text-align: center;
 }


 QProgressBar::chunk {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
    width: 20px;
 }

QFrame, QLabel, QToolTip {
   color: #e5e5e5;
   /*border: 1px solid #141414;*/
}

QFrame[style=&quot;plain&quot;] {
    border: 100px solid pink;
}

QAbstractButton {
   padding: 3px;
   border: 1px solid #141414;
   border-radius: 4px;
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #3c3c3c, stop: 1 #262627);
    color: #e8e8e8;
}
QAbstractButton::hover {
   background-color: #5b5a5b;
}

QAbstractButton::disabled {
    color: #262627;
}

QAbstractButton::checked {
    background-color: #1c1c1c;
}

 QTabWidget::pane { /* The tab widget frame */
     border-top: 1px solid #141414;
     position: absolute;
     top: -0.5em;
 }

 QTabWidget::tab-bar {
     alignment: center;
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #3c3c3c, stop: 1 #262627);

    border: 1px solid #141414;

     /*border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; same as the pane color */
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
     color: #e8e8e8;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: #5b5a5b;
 }

 QTabBar::tab:selected {
     border-color: #5b5a5b;
     color: #141414;
     /*border-bottom-color: #C2C7CB; same as pane color */
 }

 QSlider::groove:horizontal {
 border: 1px solid #141414;
     height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #3c3c3c, stop: 1 #262627);
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #3c3c3c, stop: 1 #262627);     border: 1px solid #5c5c5c;
     width: 18px;
     /*margin: -2px 0;  handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }

  QScrollBar:vertical {
      border: 0px solid transparent;
      background: transparent;
      width: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:vertical {
      background: #e8e8e8;
      min-height: 20px;
      border: 2px solid #e8e8e8;
      border-radius: 2px;
  }
  QScrollBar::add-line:vertical {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }

  QScrollBar::sub-line:vertical {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
      border: 0px solid grey;
      width: 0px;
      height: 0px;
      background: white;
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }



  QScrollBar:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:horizontal{
      background: #e8e8e8;
      min-height: 20px;
      border: 2px solid #e8e8e8;
      border-radius: 2px;
  }
  QScrollBar::add-line:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }

  QScrollBar::sub-line:horizontal {
      border: 0px solid transparent;
      background: transparent;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
      border: 0px solid grey;
      width: 0px;
      height: 0px;
      background: white;
  }

  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }





QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 50px;
border-radius: 4px;
}

QSlider::groove:vertical {
border: 1px solid #bbb;
background: white;
width: 50px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #444, stop: 1 #bbb);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbb, stop: 1 #555);
border: 1px solid #777;
height: 50px;
border-radius: 4px;
border-top-left-radius: 4px;
border-top-right-radius: 0px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 0px;
}

QSlider::sub-page:vertical {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #bbb, stop: 1 #444);
background: qlineargradient(x1: 0, y1: 1, x2: 1, y2: 0.2,
    stop: 0 #555, stop: 1 #bbb);
border: 1px solid #777;
width: 50px;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}

QSlider::add-page:horizontal {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #bbb, stop: 1 #444);
background: qlineargradient(x1: 0, y1: 1, x2: 1, y2: 0.2,
    stop: 0 #555, stop: 1 #bbb);
border: 1px solid #777;
height: 50px;
border-top-left-radius: 0px;
border-top-right-radius: 4px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 4px;
}

QSlider::add-page:vertical {
background: qlineargradient(x1: 0, y1: 1,    x2: 0, y2: 0,
    stop: 0 #bbb, stop: 1 #444);
background: qlineargradient(x1: 1, y1: 1, x2: 0, y2: 0.2,
    stop: 0 #555, stop: 1 #bbb);
border: 1px solid #777;
width: 50px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #333, stop:1 #777);
border: 1px solid #777;
width: 53px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 10px;
margin: 2px;
}

QSlider::handle:vertical {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #333, stop:1 #777);
border: 1px solid #777;
height: 53px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 10px;
margin: 2px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::sub-page:vertical:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="modeContainer" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_snarky">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;You may find the top-right menu rather useful.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_topL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="4">
       <widget class="QToolButton" name="toolButton_midL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Mid</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QToolButton" name="toolButton_lowL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButton_volL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Vol</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="toolButton_hiL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Hi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButton_snapL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>snap</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="playL">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3" colspan="4">
       <widget class="QWidget" name="widget_waveformL" native="true">
        <property name="styleSheet">
         <string notr="true">     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
border-radius: 4px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_trackL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>- no track -</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_topR">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton_volR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Vol</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButton_hiR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Hi</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="toolButton_midR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Mid</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="toolButton_lowR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_snapR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>snap</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButton_playR">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>39</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="3" colspan="4">
       <widget class="QWidget" name="widget_waveformR" native="true">
        <property name="styleSheet">
         <string notr="true">     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #3c3c3c, stop: 1 #262627);
border-radius: 4px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_trackR">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>- no track -</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="tmpWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
